archeo/inc/patterns/layered-images-with-headline.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Layered images with headline
+ */
+return array(
+	'title'      => __( 'Layered images with headline', 'archeo' ),
+	'categories' => array( 'pages' ),
+	'content'    => '<!-- wp:cover {"url":"' . esc_url( get_template_directory_uri() ) . '/assets/images/palais-du-cirque-bg.png","dimRatio":0,"overlayColor":"background","contentPosition":"center center","isDark":false,"align":"full","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0vw","left":"7vw"}}}} -->
```

Comment:
Is this pattern meant to take up the full page height i.e. `100vh`?

Reply:
Ah yeah probably, I've added this. Thanks!